index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+  <meta charset="utf-8">
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+  <link href="https://fonts.googleapis.com/css?family=Righteous" rel="stylesheet">
+  <link rel="stylesheet" href="index.css">
+  <script src="index.js" charset="utf-8"></script>
+
+  <title>Pong</title>
+</head>
+<body>
+  <div class="container">
+    <div class="mainMenu">
+      <header >
+        <div class="row">
+          <div class="col-md-12">
+            <h1>Mav Pong</h1>
+          </div>
+        </div>
+      </header>
+      <section class="row buttons">
+        <div class="col-md-6 offset-md-3">
+          <div class="row">
+            <button type="button" name="button">Instructions</button>
+          </div>
+          <div class="row">
+            <button type="button" name="button">Play</button>
+          </div>
+          <div class="row">
+            <button type="button" name="button">Score</button>
+          </div>
+        </div>
+      </section>
+    </div>
+
+  <div class="container">
+    <div class="instructions">
+      <header >
+        <div class="row">
+          <div class="col-md-12">
+            <h1>Instructions</h1>
+          </div>
+        </div>
+      </header>
+    </div>
+  </div>
+</div>
```

Comment:
what div is this </div> closing indentation is off and your div and /div may be mismatched